suite-native/app/src/navigation/RootTabNavigator.tsx

```diff
@@ -18,7 +18,7 @@ export const RootTabNavigator = () => (
         initialRouteName={RouteTabs.HomeStack}
         screenOptions={{
             headerShown: false,
-            unmountOnBlur: true,
+            unmountOnBlur: false,
```

Comment:
Much smoother user experience

Reply:
Originally it was meant for unmounting screens and the state of nested screens in stack navigator (e.g. when you go to settings, then app localisation, then to home tab and back to settings again to not show app localisation but default settings screen) but I agree that users want to probably have their navigation history keept.

Reply:
Agree, that could be problem, let's try this way now and we will see. We can easily switch it back.